source/documentors/references/quick_reference_rst.rst

```diff
@@ -4,8 +4,13 @@ Quick Reference: Writing RST
 RST is very powerful and flexible.  Below we provide a quick guide for how you
 can use it to write Open edX documentation.
 
-For more details check out the `RST Primer`_
+.. seealso::
```

Comment:
Seealso directives are usually at the bottom, and there is already one there that this link can be added to.  It feels weird to send them away from this page as soon as they have arrived to it.  I think you should just add this to the see-also section at the bottom.